### DISCUSSION TITLE
Owner is being deleted, so new repository cannot be created.

### DISCUSSION THREAD

**Discussion by criscunas** [community/community#107489] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Has anyone gotten this error when trying to create a new repository? Not sure what to do, can't find much help online.

**Comment by naveenalla3000** [comment]
every repo should have unique name. Try to create repo with another name.

**Reply by criscunas** [reply]
Yeah that doesn't fix it but thanks
